src/azul/plugins/metadata/anvil/indexer/transform.py

```diff
@@ -130,13 +132,15 @@ def field_types(cls) -> FieldTypes:
     def get_aggregator(cls, entity_type) -> EntityAggregator:
         if entity_type == 'activities':
             return ActivityAggregator()
-        if entity_type == 'biosamples':
+        elif entity_type == 'biosamples':
             return BiosampleAggregator()
-        if entity_type == 'datasets':
+        elif entity_type == 'datasets':
             return DatasetAggregator()
-        if entity_type == 'donors':
+        elif entity_type == 'donors':
             return DonorAggregator()
-        if entity_type == 'files':
+        elif entity_type == 'diagnoses':
+            return DiagnosisAggregator()
```

Comment:
Suggestion, move the test of `diagnoses` before `donors` to preserve the alphabetical order of tests